content/docs/capabilities/high-availability.mdx

```diff
@@ -122,20 +134,25 @@ Multiple replicas of Databroker or all-in-one service are only supported with [e
 
 :::
 
-## Example
+## Example: Split-Service Mode
```

Comment:
Question: should this whole section be moved into a separate guide page?

I think we're trying to emphasize that you can run a high-availability deployment in either all-in-one or split-service mode, so I don't know if it makes sense to demonstrate the split-service mode here.

Also, the example configuration below doesn't run multiple replicas of the services, does it? I'm not sure it belongs on a page about high availability at all, if we're no longer recommending that all HA deployments use split-service mode.